web/src/components/Inputs2.js

```diff
@@ -14,10 +14,10 @@ const makeInput = InputInner => {
     label,
     type,
     hideLabel,
-    className,
+    wrapperClass,
```

Comment:
This allows setting a class on the native `<input>`?  That's pretty slick.

Reply:
Also, destructuring the props ***and*** having a `...rest`...  I feel like you've done that before but it just now clicked this is happening.  Very cool.